docs/modules/installation/pages/plugin.adoc

```diff
@@ -8,24 +8,25 @@ If you do not have an existing Grafana instance, see xref:package.adoc[].
 
 == Install via web UI
 
-Follow these steps to install the {page-component-title} plugin using the Grafana UI:
+Follow these steps to install the plugin using the Grafana UI:
 
-. Navigate to your Grafana dashboard.
-. Click the *Configuration* symbol and choose *Plugins*.
+
+. Navigate to the home page of your Grafana instance.
+. In the left navigation menu, click menu:Configuration[Plugins].
 . Change the *State* to *All*.
-. Search for "OpenNMS" and select the *OpenNMS Helm* plugin.
+. Search for "OpenNMS" and select *OpenNMS Plugin for Grafana*.
 . Click *Install*.
 
 == Install via CLI
 
-You can the `grafana-cli` tool to install {page-component-title}:
+You can the `grafana-cli` tool to install the {page-component-title}:
 
 [source, console]
```

Comment:
I discovered yesterday that "console" isn't recognized as a syntax highlighter, so I'm updating these to "shell" as I come across them.

```suggestion
[source, shell]
```